Makefile

```diff
@@ -1,71 +1,87 @@
-.PHONY: generate
-generate:
-generate: generate/migrations
-generate: generate/sqlc
+.DEFAULT_GOAL := help
 
 .PHONY: db/reset
-db/reset: ## drop, create, and migrate dev and test databases
+db/reset: ## Drop, create, and migrate dev and test databases
 db/reset: db/reset/dev
 db/reset: db/reset/test
 
 .PHONY: db/reset/dev
-db/reset/dev: ## drop, create, and migrate dev database
+db/reset/dev: ## Drop, create, and migrate dev database
 	dropdb river_dev --force --if-exists
 	createdb river_dev
 	cd cmd/river && go run . migrate-up --database-url "postgres://localhost/river_dev"
 
 .PHONY: db/reset/test
-db/reset/test: ## drop, create, and migrate test databases
+db/reset/test: ## Drop, create, and migrate test databases
 	go run ./internal/cmd/testdbman reset
 
+.PHONY: generate
+generate: ## Generate generated artifacts
+generate: generate/migrations
+generate: generate/sqlc
+
 .PHONY: generate/migrations
-generate/migrations: ## sync changes of pgxv5 migrations to database/sql
+generate/migrations: ## Sync changes of pgxv5 migrations to database/sql
 	rsync -au --delete "riverdriver/riverpgxv5/migration/" "riverdriver/riverdatabasesql/migration/"
 
 .PHONY: generate/sqlc
-generate/sqlc:
+generate/sqlc: ## Generate sqlc
 	cd riverdriver/riverdatabasesql/internal/dbsqlc && sqlc generate
 	cd riverdriver/riverpgxv5/internal/dbsqlc && sqlc generate
 
+# Looks at comments using ## on targets and uses them to produce a help output.
+.PHONY: help
+help: ALIGN=22
+help: ## Print this message
+	@awk -F '::? .*## ' -- "/^[^':]+::? .*## /"' { printf "'$$(tput bold)'%-$(ALIGN)s'$$(tput sgr0)' %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
```

Comment:
This is kind of a minor nicety that I've been putting in a few projects. Just lets you run `make help` and get this:

```
$ make help
db/reset               Drop, create, and migrate dev and test databases
db/reset/dev           Drop, create, and migrate dev database
db/reset/test          Drop, create, and migrate test databases
generate               Generate generated artifacts
generate/migrations    Sync changes of pgxv5 migrations to database/sql
generate/sqlc          Generate sqlc
help                   Print this message
lint                   Run linter (golangci-lint) for all submodules
test                   Run test suite for all submodules
tidy                   Run `go mod tidy` for all submodules
update-mod-go          Update `go`/`toolchain` directives in all submodules to match `go.work`
update-mod-version     Update River packages in all submodules to $VERSION
verify                 Verify generated artifacts
verify/migrations      Verify synced migrations
verify/sqlc            Verify generated sqlc
```